content/docs/integrations/bitbucket/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Integrating with Bitbucket
+publishedDate: '2022-03-28T14:00:00.0Z'
+description: How to give Roadie the access it needs to your Bitbucket repositories.
+---
+
+## Introduction
+
+This tutorial will guide you through the steps required to connect Roadie to your Bitbucket repositories.
+
+## Step 1: Save Bitbucket integration config in Roadie
+
+By adding a Bitbucket integration you will allow Roadie to access the YAML metadata files that Backstage needs to operate.
+
+Roadie supports Bitbucket configuration on both Bitbucket Cloud and self-hosted Bitbucket server. The authentication methods supported are either with Token or a Username + Application Password authentication. Username + Application Password is preferred authentication method since it provides more granular options to control the access to give to the integration. If you are connecting to a Bitbucket server Username + Application Password is the only supported connection method. 
```

Comment:
```suggestion
Roadie supports Bitbucket configuration on both Bitbucket Cloud and self-hosted Bitbucket server. The authentication methods supported are either with Token or a Username + Application Password authentication. Username + Application Password is the preferred authentication method since it provides more granular options to control the access to give to the integration. If you are connecting to a Bitbucket server Username + Application Password is the only supported connection method. 
```